src/lando/jinja2.py

```diff
@@ -0,0 +1,15 @@
+from django.templatetags.static import static
```

Comment:
What else is going to go in this module? Could we just put this in `settings.py`?

Reply:
> What else is going to go in this module? Could we just put this in `settings.py`?

This is the typical way of adding a Jinja environment. Basically all the filters and custom functionality we added in the legacy repo will eventually go in here. This doesn't belong in the settings module.